tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -903,6 +903,51 @@ def test__get_collection_with_x_approximate_distribution_none__OK(self):
         self.assertIsNone(res.json["datasets"][0]["x_approximate_distribution"])
 
 
+class TestGetCollectionVersionID(BaseAPIPortalTest):
+    def test_get_collection_version_ok(self):
+        first_version = self.generate_published_collection()
+        revision = self.generate_revision(first_version.collection_id)
+        self.business_logic.publish_collection_version(revision.version_id)
+        res = self.app.get(
+            f"/curation/v1/collection_versions/{first_version.version_id}", headers=self.make_owner_header()
+        )
+        self.assertEqual(200, res.status_code)
+        self.assertEqual(res.json["collection_version_id"], first_version.version_id.id)
+        # test correct dataset explorer url is used
+        explorer_url = res.json["datasets"][0]["explorer_url"]
+        self.assertTrue(explorer_url.endswith(f"{first_version.datasets[0].version_id.id}.cxg/"))
+
+        res = self.app.get(f"/curation/v1/collection_versions/{revision.version_id}", headers=self.make_owner_header())
+        self.assertEqual(200, res.status_code)
+        self.assertEqual(res.json["collection_version_id"], revision.version_id.id)
+        # test correct dataset explorer url is used
+        explorer_url = res.json["datasets"][0]["explorer_url"]
+        self.assertTrue(explorer_url.endswith(f"{revision.datasets[0].version_id.id}.cxg/"))
+
+    def test_get_collection_version_403(self):
+        with self.subTest("Query endpoint with incorrect ID"):
+            res = self.app.get(
+                f"/curation/v1/collection_versions/{str(uuid.uuid4())}", headers=self.make_owner_header()
+            )
+            self.assertEqual(403, res.status_code)
+        with self.subTest("Query endpoint with Canonical ID"):
+            collection = self.generate_published_collection()
+            res = self.app.get(
+                f"/curation/v1/collection_versions/{collection.collection_id}", headers=self.make_owner_header()
+            )
+            self.assertEqual(403, res.status_code)
+        with self.subTest("Query endpoint with non-UUID"):
+            res = self.app.get("/curation/v1/collection_versions/bad-input-id", headers=self.make_owner_header())
+            self.assertEqual(403, res.status_code)
+        with self.subTest("Collection Version is part of tombstoned Collection"):
+            collection = self.generate_published_collection()
+            self.business_logic.tombstone_collection(collection.collection_id)
+            res = self.app.get(
+                f"/curation/v1/collection_versions/{collection.version_id}", headers=self.make_owner_header()
+            )
+            self.assertEqual(403, res.status_code)
```

Comment:
Add sub test to check that querying for an published revision via this endpoint fails

Reply:
do you mean unpublished revision? 

Assuming yes, does that mean we don't want the capability URL principle to be applied here?

Reply:
> do you mean unpublished revision?

Yes 😇 

> Assuming yes, does that mean we don't want the capability URL principle to be applied here?

No. The id of an **_unpublished_** revision should be treated as a canonical id from a user perspective. That means, while it really is a version id on our end, users should only see it work in places where a canonical id works: 
`/collections/{collection_id}...` ✅ 
`/collection_versions/{collection_version_id}` ⛔